controllers/api.controller.js

```diff
@@ -0,0 +1,7 @@
+const fs = require("fs/promises");
+
+exports.endpointsInfo = (req, res, next) => {
+  fs.readFile("./endpoints.json", "utf-8").then((data) => {
+    res.status(200).send(JSON.parse(data));
+  });
+};
```

Comment:
This is a good solution but similar to the above you can also require in json files directly and access the object here without needing to use the fs library.